### ISSUE TITLE
Duplicative video button titles fail accessibility guidelines

### ISSUE THREAD

**Issue by anilnatha** [MAAP-Project/ZenHub#418]
There are two slides in the carousel of the homepage that have buttons labeled "Start the Video".  For screen reader users this description is too generic.

Suggested resolution:

> Make the link text unique for each target page (which helps all users) or add an aria-label or aria-describedby to the link (which helps screen reader users).

**Comment by anilnatha** [comment]
After reviewing the SmartSlider plugin features, which is what powers the carousel on the homepage, the plugin does not provide aria fields to update the [button layers](https://smartslider.helpscoutdocs.com/article/1834-button-layer) with duplicate labels that SortSite reported on. It also doesn't provide the ability as far as I can tell to add custom attributes. I've notified the makers of SmartSlider of this shortcoming in hopes they can add the ability to edit aria attributes in the future.

I'm unsure of how to proceed with this at the moment short of replacing the entire slider component which could introduce other accessibility issues.

**Comment by oddes** [comment]
@anilnatha 
these are not ready for QA. Moving them back.

**Comment by anilnatha** [comment]
I reached out to the maintainer of this plugin and provided them the details in hopes they can someday fix this issue.

**Comment by anilnatha** [comment]
The maintainer of the plugin has uncovered an issue with SortSite that I was not aware of. SortSite appears to be generating false positives in situations where an element has a property of `display: none`. In those cases, a screen reader shouldn't be reading those elements as it's not being displayed. This issue can be closed as I have verified with other online accessibility tools that it is not being reported.
